core/src/main/java/org/geysermc/geyser/translator/protocol/bedrock/entity/player/BedrockActionTranslator.java

```diff
@@ -201,35 +218,34 @@ public void translate(GeyserSession session, PlayerActionPacket packet) {
                 }
 
                 Vector3f vectorFloat = vector.toFloat();
-                LevelEventPacket continueBreakPacket = new LevelEventPacket();
-                continueBreakPacket.setType(LevelEvent.PARTICLE_CRACK_BLOCK);
-                continueBreakPacket.setData((session.getBlockMappings().getBedrockBlockId(breakingBlock)) | (packet.getFace() << 24));
-                continueBreakPacket.setPosition(vectorFloat);
-                session.sendUpstreamPacket(continueBreakPacket);
 
                 // Update the break time in the event that player conditions changed (jumping, effects applied)
                 LevelEventPacket updateBreak = new LevelEventPacket();
                 updateBreak.setType(LevelEvent.BLOCK_UPDATE_BREAK);
                 updateBreak.setPosition(vectorFloat);
-                double breakTime = BlockUtils.getSessionBreakTime(session, BlockState.of(breakingBlock).block()) * 20;
 
+                BlockState blockState = BlockState.of(breakingBlock);
```

Comment:
`breakingBlock` or `breakingBlockState`

Reply:
I renamed it to breakingBlockState in latest commit.